### PR TITLE
feat(Card): add pentaho plus styles

### PR DESCRIPTION
**Pull request by plagoa**

### CODE REVIEW

File: packages/core/src/Card/stories/ExpandableCard.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      >
+        <HvTypography>This is my hidden content</HvTypography>
+      </HvCardContent>
+
+      <div
+        style={{ display: "flex", justifyContent: "center", paddingBottom: 0 }}
+      >
+        <HvIconButton
+          title={isOpen ? "Collapse" : "Expand"}
+          onClick={() => setIsOpen(!isOpen)}
```
Comment (MEsteves22) [inline packages/core/src/Card/stories/ExpandableCard.tsx:53]: Looking at [these docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded), I think we need to use the `aria-expanded` prop on the button and also use `aria-controls` (and set an `id` to the `HvCardContent`) 🙏

File: packages/styles/src/themes/pentahoPlus.ts [hunk 2]
```diff
@@ -730,6 +730,60 @@ const pentahoPlus = makeTheme((theme) => ({
         },
       },
     },
+    HvCard: {
```
Comment (MEsteves22) [inline packages/styles/src/themes/pentahoPlus.ts:733]: Shouldn't we remove the Pentaho card from the Pentaho package?

File: packages/styles/src/themes/pentahoPlus.ts [hunk 3]
```diff
@@ -730,6 +730,60 @@ const pentahoPlus = makeTheme((theme) => ({
...
+      classes: {
+        root: {
+          outline: `1px solid ${theme.palette.slate[200]}`,
```
Comment (MEsteves22) [inline packages/styles/src/themes/pentahoPlus.ts:736]: I think we should put a `TODO` on the colors from palette we are using to not forget we need to use a token when available.

**Review by MEsteves22 — COMMENTED** [review]
LGTM 👌🏻

I just have the following comments:

**Comment by plagoa** [comment]
@MEsteves22 PR reviewed with feedback

**Review by zettca — APPROVED** [review]
Can we mark the removing Card commit as a breaking change, so that users (maybe none yet) get better visibility, and for us to test the versioning + release messaging

**Pull request merged**